.circleci/config.yml

```diff
@@ -9,10 +9,11 @@ jobs:
           ALLOWED_HOSTS: localhost
           DATABASE_URL: postgresql://root@localhost/circle_test?sslmode=disable
           BIRDBATH_REQUIRED: false
-      - image: circleci/postgres:11.5
+      - image: circleci/postgres:12.3
```

Comment:
```suggestion
      - image: circleci/postgres:12.17
```

**suggestion** The version on production is using 12.17 so maybe the docker-compose.yml file DB version could be updated too.